### ISSUE TITLE
Episode 2: Introduction to Keras

### ISSUE THREAD

**Issue by dafnevk** [carpentries-incubator/deep-learning-intro#43]
## Learning objectives 
- Explain what a classification task is 
- Use one-hot-encoding to prepare data for classification in Keras 
- Describe a fully connected layer 
- Implement a fully connected layer with Keras 
- Use Keras to train a small fully connected network on prepared data 
- Plot the loss curve of the training process 

## Questions: 
- What happens to the number of parameters if you add another layer? 
- How do I compose a Deep Neural Network using Keras? 
- How do I train this network on a dataset 
- How do I get insight into learning process 
- How do I measure the performance of the network 

This episode will probably use the IRIS dataset

**Comment by psteinb** [comment]
I think, #66 needs to align to this.

**Comment by florian-huber** [comment]
With @dafnevk and @bpmweel we just discussed that we could also use the weather dataset here (instead of the penguins).
One idea on this is to introduce a classification label: barbecue day --> yes/no
